docs/extensions/extension_example.md

```diff
@@ -212,7 +215,7 @@ await Deno.writeTextFile(
 When replacing the `console.log` with this function call and executing our
 extension, you'll run into the following error:
 
-```
+```console
 ❗ Error: Execution failed caused by: Error: Requires write access to "./duplicates.txt"
 ```
```

Comment:
I think this should be text too, no reason to color it blue (at least that's what GitHub does).

Reply:
Roger that...will do.